From: Russ Allbery <eagle@eyrie.org>
Date: Sun, 29 Dec 2013 12:29:44 -0800
Subject: Add explicit Requires from lbcd.service to lbcd.socket

Since there's no reason to ever run in a mode other than socket
activation, add an explicit Requires so that lbcd.socket is always
started first.  Also add the Emacs conf-mode comment to the
lbcd.socket file.
---
 systemd/lbcd.service.in | 1 +
 systemd/lbcd.socket     | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/systemd/lbcd.service.in b/systemd/lbcd.service.in
index 392544b..d1d2ed3 100644
--- a/systemd/lbcd.service.in
+++ b/systemd/lbcd.service.in
@@ -4,6 +4,7 @@
 Description=responder for load balancing
 Documentation=man:lbcd(8)
 Documentation=http://www.eyrie.org/~eagle/software/lbcd/
+Requires=lbcd.socket
 
 [Service]
 Type=simple
diff --git a/systemd/lbcd.socket b/systemd/lbcd.socket
index 8227551..725c8e8 100644
--- a/systemd/lbcd.socket
+++ b/systemd/lbcd.socket
@@ -1,3 +1,5 @@
+# Socket configuration for lbcd service.  -*- conf -*-
+
 [Unit]
 Description=lbcd listening socket
 Documentation=man:lbcd(8)
